reporter/reporter.py

```diff
@@ -157,41 +150,6 @@ def add_geodata(entity):
         geocoding.add_location(entity, cache=cache)
 
 
-def query_1T1E1A(attrName, entityId, type=None, aggrMethod=None,
-                 aggrPeriod=None, options=None, fromDate=None, toDate=None,
-                 lastN=None, limit=None, offset=None):
-    """
-    See /entities/{entityId}/attrs/{attrName} in API Specification
-    quantumleap.yml
-    """
-    DB_HOST = os.environ.get('CRATE_HOST', 'crate')
-    DB_PORT = 4200
-    DB_NAME = "ngsi-tsdb"
-
-    from translators.crate import CrateTranslator
-    TIME_INDEX = CrateTranslator.TIME_INDEX_NAME
-
-    with CrateTranslator(DB_HOST, DB_PORT, DB_NAME) as trans:
-        attr_names = [TIME_INDEX, attrName]
-        vals = trans.query(attr_names=attr_names, entity_type=type,
-                           entity_id=entityId)
-    res = {
-        'data': {
-            'entityId': entityId,
-            'attrName': attrName,
-            'index': [str(v[TIME_INDEX]) for v in vals],
-            'values': [v[attrName]['value'] for v in vals]
-        }
-    }
-    return res
-
-# Before implementing the rest of these methods, move query methods to a
-# different module.
-
-def query_1T1E1A_value():
-    raise NotImplementedError
-
-
 def query_1T1ENA():
     raise NotImplementedError
```

Comment:
i saw code for query_1T1ENA is this still valid?

Reply:
good catch, removed.